packages/tempus/src/Tempus_StepperFactory.hpp

```diff
@@ -383,6 +390,33 @@ class StepperFactory
   // ---------------------------------------------------------------------------
   // Create individual Steppers.
 
+  Teuchos::RCP<StepperSubcycling<Scalar> >
+  createStepperSubcycling(
+    const Teuchos::RCP<const Thyra::ModelEvaluator<Scalar> >& model,
+    Teuchos::RCP<Teuchos::ParameterList> stepperPL)
+  {
+    auto stepper = Teuchos::rcp(new StepperSubcycling<Scalar>());
+
+    TEUCHOS_TEST_FOR_EXCEPTION(stepperPL != Teuchos::null, std::logic_error,
+      "Error - Construction of StepperSubcycling with a ParameterList\n"
+      "is not implemented yet!\n");
+
+    if (stepperPL != Teuchos::null) {
+      setStepperValues(stepper, stepperPL);
+      //setStepperSubcyclingValues(stepper, stepperPL);
+    }
+    //else {
+    //  integrator->setTempusParameterList(Teuchos::null);
+    //}
```

Comment:
Is this path ever triggered?

Reply:
For the subcycling, we do not have any tests for ParameterList construction and nobody should be using it yet.  So I did not code up this path.